scripts/tests/general.spec.js

```diff
@@ -0,0 +1,31 @@
+const Helper = require('hubot-test-helper');
```

Comment:
The test for a file should live right next to the file so instead of this file being `scripts/tests/general.spec.js`, it should just be `scripts/general.spec.js`.

Reply:
No, but I see what you're saying. Maybe we could move the tests to another folder called `tests/` or something in the project root then? Having the tests nested under the files feels weird.